src/ui/component/fileListTrending/index.js

```diff
@@ -0,0 +1,18 @@
+import { connect } from 'react-redux';
+import { selectFollowedTags, doFetchByTags, selectTrendingUris, selectFetchingTrending } from 'lbry-redux';
+import DiscoverPage from './view';
+
+const select = state => ({
+  followedTags: selectFollowedTags(state),
+  trending: selectTrendingUris(state),
```

Comment:
similar to my feedback on the lbry-redux PR, I think using the name trending repeatedly is a mild mistake, because it does things beyond trending

I would call the entire component FileListDiscover and call the above value just `uris` or something like this